src/java.base/share/classes/java/lang/classfile/constantpool/ConstantPoolBuilder.java

```diff
@@ -474,10 +474,13 @@ default StringEntry stringEntry(String value) {
     }
 
     /**
-     * {@return A {@link ConstantValueEntry} descripbing the provided
+     * {@return A {@link ConstantValueEntry} describing the provided
      * Integer, Long, Float, Double, or String constant}
+     * The returned consant is suitable as a value for the
```

Comment:
This information is already available on `ConstantValueEntry`. Since we already link to it on the return type, I feel like this is a redundancy.

However, `ConstantValueEntry::constantValue()` is missing a link to this API; would be nice if you can add that. And `ConstantValueEntry` is missing a link to `Attributes::constantValue()` too.